libs/rpc/drivers/extension-content-driver.js

```diff
@@ -5,42 +5,73 @@
  */
 var ExtensionContentDriver = function () {
```

Comment:
This is queuing is not just because of switching tabs, it will queue actions when page is not fully loaded or content/background script is still not ready or connected.
I'm not sure how much is this case rare, even if it is we need to make software that covers all cases.